### PR TITLE
Validate eve.json output /v1

### PR DESCRIPTION
**Pull request by TharushiJay**
Validates eve.json output using a JSON schema.

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/1369

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
+    "type" : "object",
+     "properties": {
+        "timestamp": {
+            "type": "string",
+	        "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+-\\d+$",
```
Comment (jasonish) [inline schema.json:6]: Need to handle a "+" in the timezone offset as well for timestamps like:
```2020-07-30T14:53:17.457474+0000```
this should probably do it:
```
            "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+[+\\-]\\d+$",
```

File: schema.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            "optional" : false
+	                },
```
Comment (jasonish) [inline schema.json:8]: minor: indenting seems off here.  vscode shift+ctrl+i seems to clean it up nicely.

**Comment by jasonish** [comment]
Nice work, this will be very useful.  I'd like to see some modifications to the output:
- Don't output anything except errors.  And when there is an error, output the test name, etc.
- If `-v` is provided, then print the "===> test_name: OK/FAIL" type thing.
- Include a summary at the end of how many eve.json files were checked and pass fail count.

**Review by jasonish — CHANGES_REQUESTED** [review]

**Comment by TharushiJay** [comment]
Changes continued in #428

**Pull request closed without merging**